packages/StepTracker/__tests__/StepTracker.spec.jsx

```diff
@@ -37,35 +38,24 @@ describe('<StepTracker />', () => {
     expect(stepTracker.find('[data-testid="step-0"]')).toContainReact(
       <Icon symbol="checkmark" size={16} variant="inverted" />
     )
-    expect(stepTracker.find('[data-testid="singleStepContainer-0"]')).toHaveProp(
-      'data-isactive',
-      true
-    )
+
+    expect(stepTracker).toMatchSnapshot()
```

Comment:
Is there a way to test this other way than snapshots? All these snapshot matchers made the actual snapshot file too big which will make it harder to read later when it breaks.